Specs/Core/GregorianDateSpec.js

```diff
@@ -0,0 +1,300 @@
+import { GregorianDate } from "../../Source/Cesium.js";
+
+describe("Core/GregorianDate", function () {
+  describe("Positive scenarios", function () {
```

Comment:
A lot of our tests don't use nested `describes` unless they get very long, but it's fine to leave this in. I would just be specific -- instead of "positive scenarios", say "valid parameters." Instead of "negative scenarios," "invalid parameters."